ci-operator/step-registry/ipi/conf/azure/multidisk/swap/ipi-conf-azure-multidisk-swap-commands.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+function swap_machineconfig_generate(){
```

Comment:
Can we merge this step to `ipi-conf-azure-multidisk` step ?

Reply:
Done